app-navigation/navigation.md

```diff
@@ -62,8 +67,8 @@ The Request content contains the method buttons - GET, POST, PUT, PATCH and DELE
 
 Three methods - POST, PUT and PATCH - allow you to set the **Body**:
 
-* JSON - The JSON body automatically saves and checks for valid syntax.
-* RAW - The RAW request body can be saved using the _Save Icon_.
+- JSON - The JSON body automatically saves and checks for valid syntax.
```

Comment:
I'd be a bit more specific here about how the saving of the json input works: If there are any changes in the json editor while you are typing you can use the save icon to save the content. Otherwise it'll save any valid content when you click outside of the editor.